x-pack/plugins/ml/public/embeddables/job_creation/common/get_flyout_session.tsx

```diff
@@ -8,27 +8,27 @@
 import React from 'react';
 import { takeUntil, distinctUntilChanged, skip } from 'rxjs/operators';
 import { from } from 'rxjs';
-import type { Embeddable } from '@kbn/lens-plugin/public';
-import type { CoreStart } from '@kbn/core/public';
-import type { SharePluginStart } from '@kbn/share-plugin/public';
-import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
-import type { LensPublicStart } from '@kbn/lens-plugin/public';
-
 import {
   toMountPoint,
   wrapWithTheme,
   KibanaContextProvider,
 } from '@kbn/kibana-react-plugin/public';
+import type { SharePluginStart } from '@kbn/share-plugin/public';
+import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
+import type { CoreStart } from '@kbn/core/public';
+import type { LensPublicStart } from '@kbn/lens-plugin/public';
+import type { MapEmbeddable } from '@kbn/maps-plugin/public';
+import type { Embeddable } from '@kbn/lens-plugin/public';
 
-import { getMlGlobalServices } from '../../application/app';
-import { LensLayerSelectionFlyout } from './lens_vis_layer_selection_flyout';
+import { getMlGlobalServices } from '../../../application/app';
 
-export async function showLensVisToADJobFlyout(
-  embeddable: Embeddable,
+export function getFlyoutSession(
+  FlyoutComponent: any,
```

Comment:
should be `React.ReactElement`

Reply:
Updated to correct type in 8da1fadcaca00341a6a40c9abda84d799ceddd95